packages/mapviewer/src/store/modules/ui.store.js

```diff
@@ -183,6 +183,14 @@ export default {
          * @type Boolean
          */
         showDragAndDropOverlay: false,
+
+        /**
+         * Flag telling if we should hide the UI elements in the embed viewer. Zoom buttons, 3d
+         * button and the `view on geoadmin link`
+         *
+         * @type Boolean
+         */
```

Comment:
Might make sense to add in there that this is specific for GeoPlateformSchweiz, and that we shouldn't disclose this param (as it will soon be obsolete and GPS will replace the use of iframe with the new JS API as soon as it's available)

Reply:
good idea